test/others/ns_ext/run.sh

```diff
@@ -54,19 +58,35 @@ ino2=$(ls -iL $MNT2 | awk '{ print $1 }')
 exec 33< $MNT1
 exec 34< $MNT2
 $CRIU dump -v4 -t $pid -o dump.log -D images --external $NS[$ino]:test_ns --external $NS[$ino2]:test_ns2
+RESULT=$?
 cat images/dump.log | grep -B 5 Error || echo ok
+[ "$RESULT" != "0" ] && {
+	echo "CRIU dump failed"
+	echo FAIL
+	exit 1
+}
+
 $CRIU restore -v4 -o restore.log -D images --inherit-fd fd[33]:test_ns --inherit-fd fd[34]:test_ns2 -d
+RESULT=$?
 cat images/restore.log | grep -B 5 Error || echo ok
+[ "$RESULT" != "0" ] && {
+	echo "CRIU restore failed"
+	echo FAIL
+	exit 1
+}
+
 if [[ "$NS" == "pid" ]]; then
 	pid=$(pgrep -f "^sh _run.sh pidfile2 $RND2")
 fi
 new_ino=$(ls -iL /proc/$pid/ns/$NS | awk '{ print $1 }')
 new_ino2=$(ls -iL /proc/$pid2/ns/$NS | awk '{ print $1 }')
 [ "$ino" != "$new_ino" ] && {
+	echo "Inode of new NS is different"
 	echo FAIL
 	exit 1
 }
 [ "$ino2" != "$new_ino2" ] && {
+	echo "Inode of new NS is different"
 	echo FAIL
 	exit 1
 }
```

Comment:
@adrianreber Could you add `make -C test/others/ns_ext run` in `scripts/travis/travis-tests`?

Reply:
That is already there https://github.com/checkpoint-restore/criu/blob/criu-dev/scripts/travis/travis-tests#L215

Reply:
you are right, thanks